tests/wdio.conf.js

```diff
@@ -227,6 +228,24 @@ const baseConfig = {
    */
   before: async function () {
     await browser.url('/');
+    await browser.cdp('Log', 'enable');
+    await browser.cdp('Runtime', 'enable');
+    browser.on('Runtime.consoleAPICalled', (data) => {
+      if (data &&(data.type === 'error' || data.type ==='warning' )) {
+        const logEntry = `log Event: ${JSON.stringify(data.args)}\n`;
+        console.log(`~~~~~ log event ~~~~~`);
+        console.log(logEntry);
+        console.log(`~~~~~ log event end ~~~~~`);
+        fs.appendFileSync(browserLogPath, logEntry);
```

Comment:
Maybe the duplicates probably happen because the same log event fires `Runtime.consoleAPICalled` AND `Log.entryAdded` ?

Reply:
Sadly it was occurring when before I had implemented the `Runtime.consoleAPICalled` .

Reply:
And is the event indeed fired twice? You get double logs in the test console as well?

Reply:
yeah it's doubled in both. I'm not sure why.  I'll dig a bit more.

Reply:
A bug?

https://github.com/webdriverio/webdriverio/issues/6347